ods-devenv/packer/CentOS2ODSBox.json

```diff
@@ -20,6 +20,9 @@
             "ssh_username": "{{user `username`}}",
             "ssh_password": "{{user `password`}}",
             "ami_name": "ODS in a Box {{user `ods_branch` | clean_resource_name}} {{isotime | clean_resource_name}}",
+            "ami_groups": [
```

Comment:
This change was tried in master and broke the build. 

Let us hold on on this until we have a working implementation branch `feature/ods-devenv`